src/utils/apiIntegrity.js

```
@@ -0,0 +1,803 @@
+      requestConfig.headers[INTEGRITY_HEADERS.appName] = getAppName();
+
+      const headers = await getIntegrityHeaders(requestConfig);
+      if (Object.keys(headers).length) {
+        Object.assign(requestConfig.headers, headers);
+      }
+      return requestConfig;
+    },
+    error => Promise.reject(error),
+  );
+
+  responseInterceptorId = dokApi.interceptors.response.use(
```

ESLint

[error] 'responseInterceptorId' is not defined.

(no-undef)